openjdk-testing/src/main/java/org/conscrypt/testing/TestClient.java

```diff
@@ -39,8 +37,8 @@ public TestClient(SSLSocket socket) {
     public void start() {
         try {
             socket.startHandshake();
-            input = new BufferedInputStream(socket.getInputStream());
-            output = new BufferedOutputStream(socket.getOutputStream());
+            input = socket.getInputStream();
```

Comment:
I thought that was the point of message size. If not, then it's a good idea to parameterize by buffer size as well.